### ISSUE TITLE
Why `lsp_cmd` has no sane default?

### ISSUE THREAD

**Issue by maximbaz** [ul/kak-lsp#146]
> \# NOTE This is the only option without a sane default value.
> \# You must set it manually when sourcing lsp.kak directly.
> \# Sourcing via `kak-lsp --kakoune` fills template automatically.
> decl str lsp_cmd '{{cmd}} {{args}}'

Why? Why can't `lsp_cmd` default to finding `kak-lsp` executable in `$PATH`? For users of `kak-lsp` distributed via package managers I assume it would be more natural to source `rc/lsp.kak` instead of doing `eval %sh` magic, but having to set full path to `lsp_cmd` when `kak-lsp` is already in `$PATH` is annoying.

Would be nice to default `lsp_cmd` to plain `kak-lsp` and rely on `$PATH`, and let users optionally override it with full path to a custom location of the binary.

In other words, this seems redundant:

```
set global lsp_cmd kak-lsp
```

**Comment by ul** [comment]
That sounds reasonable. We can do something like 'decl str lsp_cmd %{kak-lsp -s %val{session}}' and change Rust code to replace it in `--kakoune` output as it was doing with a template.

**Comment by maximbaz** [comment]
Thanks for the quick fix 😉
